main.tf

```diff
@@ -0,0 +1,117 @@
+resource "aws_vpc" "demo_vpc" {
+  cidr_block = "10.0.0.0/16"
+  tags = {
+    "Name" = "demo-vpc"
+  }
+}
+resource "aws_subnet" "public0" {
```

Comment:
Top-level blocks should always be separated from one another by one blank line. 

```suggestion
}

resource "aws_subnet" "public0" {
```

Reply:
Please adjust it here and for all other top-level blocks

Reply:
ok